lesson6/homework_06.py

```diff
@@ -0,0 +1,134 @@
+# Вправа 1: Проста математика
+print("\n=== ВПРАВА 1: Калькулятор ===")
+print("Створіть простий калькулятор для двох чисел і двох дій")
+print("Підтримувані операції: +, -")
+
+# Початок реалізації:
+num1 = float(input("Введіть перше число: "))
+operation = input("Введіть операцію (+, -, ): ")
+num2 = float(input("Введіть друге число: "))
+if operation == "+":
+        result = num1 + num2
+        print(f"Результат: {num1} + {num2} = {result}")
```

Comment:
```suggestion
    result = num1 + num2
    print(f"Результат: {num1} + {num2} = {result}")
```
тут і далі - відступ в 4 пробіла має бути